net/suricata6/Makefile

```diff
@@ -0,0 +1,112 @@
+#
+# Copyright (C) 2006-2015 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=suricata
+PKG_VERSION:=6.0.0-beta1
```

Comment:
There is released version 6.0.1 and you dont need to use beta version anymore.

Reply:
And they are working on 7.  I've not updated this since 6.0.1 went live